src/components/radios/index.md.njk

```diff
@@ -84,19 +84,23 @@ If one or more of your radio options is different from the others, it can help u
 
 ### Conditionally revealing content
 
-You can conditionally reveal content when the user selects a particular radio, so they only see content when it’s relevant to them.
+You can conditionally reveal a related question when the user selects a particular radio option, so they only see content when it’s relevant to them.
 
-For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
+This might make two related questions easier to answer by grouping them together. For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
 
 {{ example({group: "components", item: "radios", example: "conditional-reveal", html: true, nunjucks: true, open: false, size: "xl"}) }}
 
-Keep it simple. If you need to add a lot of content, consider showing it on the next page in the process instead.
+Keep it simple. If you need to ask a complicated question, consider showing it on the next page in the process instead.
 
-Do not add conditionally revealing content to inline radios.
+Do not add conditionally revealing content to inline radio options, such as ‘yes’ and ‘no’ options placed next to each other.
+
+Conditionally reveal questions only - do not show or hide anything that is not a question.
 
 #### Known issues
 
-Users are not always notified when conditionally revealed content is expanded or collapsed. This fails [WCAG 2.1 success criterion 4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html). We plan to fix this issue by the end of June 2021.
+Users are not always notified when a conditionally revealed question is expanded or collapsed. This fails [WCAG 2.1 success criterion 4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html).
+
+However, we found that screen reader users did not have difficulty finding and understanding follow-up questions when they’re shown - as long it’s kept simple. However, complicated follow-up questions, particularly with more than one part, did confuse users.
```

Comment:
(Sorry for this cluster of questions - it's a pity GitHub does not allow you to comment on just a specific bit of text, as opposed to the whole line.)

Does "they're" refer to the users who are being shown the questions? Or does it refer to the questions that are being shown?

Are "follow-up questions" the same things as, or different from, conditionally revealed questions?

Also, the text before the hyphen has plural terms ("questions," "they're"), but the text after the hyphen has a singular object ("as long as _it's_ kept simple"). Do we need to the content on both sides of the hyphen either wholly singular or wholly plural? Or does "they're" and "it's" refer to different things?

Reply:
No problem! Comments and amends below:

1. They're refers to the questions, but I don't see the harm if it's read as users, since both would refer to the same pint in time.
2. Yes, they are... this might be another case where trying to use simple language just confuses things when it's already called something else.
3. Yes, the singular/plural usage on this sentence is confusing. I was actively avoiding plurals in the 'good practice sentence' as much as I can, to make it clear that it's good. And plural forms of 'keep it simple' and 'kept simple' just didn't read as nicely!

```suggestion
Users are not always notified when a conditionally revealed question is expanded or collapsed. This fails [WCAG 2.1 success criterion 4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html).

However, we found that screen reader users did not have difficulty answering a conditionally revealed question - as long it’s kept simple. Users we tested with did get confused when complicated questions were conditional revealed to them, particularly questions with more than one part.
```

Reply:
Looks good! One last suggestion: in the second sentence, should it be "conditionally revealed" instead of "conditional revealed"?

Reply:
That's a good spot, cheers!